tests/test_examples.py

```diff
@@ -584,6 +584,7 @@ def _create_command_line(
                 "--use_habana",
                 "--throughput_warmup_steps 3",
                 "--save_strategy no",
+                "--trust_remote_code True",
```

Comment:
We shouldn't enable this by default for all tests but rather add it for the tests that need it.
You can add custom arguments for each test within the `extra_arguments` field of the JSON files (e.g. https://github.com/huggingface/optimum-habana/blob/22c069ee7410c0f75822dc690bec6b4d2f016cbc/tests/baselines/wav2vec2_base.json#L14).

Reply:
@regisss , I limited it to ast_finetuned_speech_commands_v2.json